Cql/Cql.Primitives/Cql.Primitives.csproj

```diff
@@ -16,6 +16,14 @@
 		<PackageReference Remove="Microsoft.CodeAnalysis.PublicApiAnalyzers" />
 	</ItemGroup>
 
+	<ItemGroup>
+		<InternalsVisibleTo Include="TemporaryTupleAssembly" Key="$(LibraryPKHash)" />
```

Comment:
What's this?

Reply:
This was needed to get the generated expressions to build, since TupleBaseType was changed to internal, but this only worked in Debug build, not Release. I changed this type back to public, so this line is no longer needed